### PR TITLE
Scala to Python - rdd folder

### PR DESCRIPTION
**Pull request by pedromb**
Converted all the scala files on the RDD folder to python

### CODE REVIEW

File: rdd/airports/AirportsByLatitudeSolution.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from pyspark import SparkContext
+from commons.Utils import Utils
+
+def splitComma(line: str):
```
Comment (jleetutorial) [inline rdd/airports/AirportsByLatitudeSolution.py:4]: Have you tried to run this program? It doesn't compile
```
  File "/Users/cwei/code/python-spark-tutorial-new/rdd/airports/AirportsByLatitudeSolution.py", line 4
    def splitComma(line: str):
                       ^
SyntaxError: invalid syntax
```

**Comment by jleetutorial** [comment]
For all the programs which print to standard output, please set the logging level to ERROR so that there is less noise in the output.

File: rdd/airports/AirportsInUsaSolution.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from pyspark import SparkContext
+from commons.Utils import Utils
+
+def splitComma(line: str):
```
Comment (jleetutorial) [inline rdd/airports/AirportsInUsaSolution.py:4]: Again, it didn't compile, I think you don't need the type.

```
  File "/Users/cwei/code/python-spark-tutorial-new/rdd/airports/AirportsInUsaSolution.py", line 4
    def splitComma(line: str):
                       ^
SyntaxError: invalid syntax
```

File: rdd/collect/CollectExample.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from pyspark import SparkContext
+
+if __name__ == "__main__":
+    sc = SparkContext("local", "collect")
```
Comment (jleetutorial) [inline rdd/collect/CollectExample.py:4]: Please set the logging level to ERROR similar to what the Scala problem does to reduce the noise of the output

File: rdd/count/CountExample.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+from pyspark import SparkContext
+
+if __name__ == "__main__":
```
Comment (jleetutorial) [inline rdd/count/CountExample.py:3]: again, set the logging level to ERROR

File: rdd/nasaApacheWebLogs/UnionLogSolutions.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
+from pyspark import SparkContext
+
+def isNotHeader(line:str):
```
Comment (jleetutorial) [inline rdd/nasaApacheWebLogs/UnionLogSolutions.py]: doesn't compile
```  File "/Users/cwei/code/python-spark-tutorial-new/rdd/nasaApacheWebLogs/UnionLogSolutions.py", line 3
    def isNotHeader(line:str):
                        ^
SyntaxError: invalid syntax```

File: rdd/reduce/ReduceExample.py [hunk 6]
```diff
@@ -0,0 +1,8 @@
+from pyspark import SparkContext
+
+if __name__ == "__main__":
```
Comment (jleetutorial) [inline rdd/reduce/ReduceExample.py:3]: set the logging level to ERROR

File: rdd/airports/AirportsByLatitudeSolution.py (hunk 1 above)
Comment (pedromb) [reply rdd/airports/AirportsByLatitudeSolution.py:4]: Hi, yes I did ran all programs. Which version of python are you running? This should work in the latest Python 3 version

Comment (jleetutorial) [reply rdd/airports/AirportsByLatitudeSolution.py:4]: sorry for the confusion. It works for Python 3. I was running python 2.7. Feel free to ignore this comment.

File: rdd/collect/CollectExample.py (hunk 3 above)
Comment (pedromb) [reply rdd/collect/CollectExample.py:4]: Hi James, some considerations about the logging level when using pyspark:
 - From the script itself, when using pyspark, we can only set the log level after starting the SparkContext, this means that logs printed when the SparkContext is starting will be printed anyway.
 - The best way to reduce the noise of the output is to configure the file log4j.properties inside spark/conf folder.
That being said, I will set the log levels to ERROR after the SparkContext starts

**Review by jleetutorial — APPROVED** [review]

**Pull request merged**